src/main/java/lambdaperson/FilteredPersons.java

```diff
@@ -0,0 +1,23 @@
+package lambdaperson;
+
+import personlist.Person;
+import java.util.Comparator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class FilteredPersons {
+
+    public static Set<Person> filterPersons(Set<Person> personList) {
```

Comment:
Should it return just names, according to the task?
 `which returns a list of persons names`